railties/lib/rails/generators.rb

```diff
@@ -218,6 +218,7 @@ def sorted_groups
         rails.delete("app")
         rails.delete("plugin")
         rails.delete("encrypted_secrets")
+        rails.delete("encryption_key_file")
```

Comment:
I take it we want `master_key` (or whatever the exact namespace is) as well?

Reply:
done